src/EverySet.elm

```diff
@@ -55,14 +42,14 @@ based on [EveryDict](http://package.elm-lang.org/packages/eeue56/elm-all-dict/la
 
 -}
 
-import EveryDict exposing (EveryDict)
+import AssocList as EveryDict exposing (Dict)
```

Comment:
I think we can get rid of the word `EveryDict` here and `import AssocList exposing (Dict)`